packages/plugin-ext/src/plugin/command-registry.ts

```diff
@@ -102,9 +102,13 @@ export class CommandRegistryImpl implements CommandRegistryExt {
     executeCommand<T>(id: string, ...args: any[]): PromiseLike<T | undefined> {
         if (this.handlers.has(id)) {
             return this.executeLocalCommand(id, ...args);
-        } else {
+        } else if (KnownCommands.transformers.hasOwnProperty(id)) {
+            // Using the KnownCommand exclusions, convert the commands manually
             return KnownCommands.map(id, args, (mappedId: string, mappedArgs: any[] | undefined) =>
```

Comment:
Should not we change `map` function instead that it uses `KnownCommands.mappings[id]` if there is no `transformers`? 

The thing is that `map` is used to create commands for code lenses, quick fixes and so on. They also should call a properly mapped command.

Reply:
I don't think there is a "should" in there: code action commands, etc. would not work with this code, right?